### PR TITLE
test elementwise leftshift on BigInt by vector of ints

### LINKED ISSUE #13832: InexactError caused by left bit-shift by a BigInt
This OK

```
julia> BigInt(2) .^ [1:6;]
6-element Array{BigInt,1}:
  2
  4
  8
 16
 32
 64
```

Not if the result is correct, there should be Array type: `Array{BigInt,1}`

```
julia> BigInt(1) .<< [1:6;]
6-element Array{Int64,1}:
  2
  4
  8
 16
 32
 64
```

therefore a large number

```
julia> BigInt(1) .<< [1:70;]
ERROR: InexactError()
 in convert at gmp.jl:188
 in .<< at arraymath.jl:120
```

### PR DESCRIPTION
**Pull request by KristofferC**
Closes #13832

### CODE REVIEW

File: test/int.jl [hunk 1]
```diff
@@ -222,3 +222,5 @@ for T in [Base.BitInteger_types..., BigInt],
     U in [Base.BitInteger_types..., BigInt]
     @test typeof(rand(U(0):U(127)) % T) === T
 end
+
```
Comment (fredrikekre) [inline test/int.jl]: `# issue #13832`? :)

Comment (KristofferC) [inline test/int.jl]: done

File: test/int.jl [hunk 2]
```diff
@@ -222,3 +222,6 @@ for T in [Base.BitInteger_types..., BigInt],
...
+# issue #13832
+@test (BigInt(1) .<< [1:70;])[end] == 1180591620717411303424
```
Comment (rfourquet) [inline test/int.jl]: Why not test also that the array is of type `BigInt`?

Comment (KristofferC) [inline test/int.jl]: Updated

**Pull request merged**